contracts/src/Utilities/ValidatorSetSig.sol

```diff
@@ -11,20 +11,39 @@ import {
 } from "@avalabs/subnet-evm-contracts@1.2.0/contracts/interfaces/IWarpMessenger.sol";
 import {ReentrancyGuard} from "@openzeppelin/contracts@4.8.1/security/ReentrancyGuard.sol";
 
+/**
+ * THIS IS AN EXAMPLE CONTRACT THAT USES UN-AUDITED CODE.
+ * DO NOT USE THIS CODE IN PRODUCTION.
+ */
+
+/**
+ * @dev Message format for the WarpMessage payload to be forwarded to the target contract
+ *
+ * validatorSetSigAddress: Address of the ValidatorSetSig contract this message is intended for
+ * targetContractAddress: Address of the contract that the payload should be forwarded to
+ * targetBlockchainID: Blockchain ID of the chain the message is intended for
+ * nonce: Unique nonce for the target contract address to provide replay protection
+ * payload: Payload to be forwarded to the target contract. Usually ABI encoded function call with parameters.
+ */
 struct ValidatorSetSigMessage {
     address validatorSetSigAddress;
     address targetContractAddress;
-    bytes32 targetBlockChainID;
+    bytes32 targetBlockchainID;
     uint256 nonce;
-    bytes txPayload;
+    bytes payload;
 }
 
 /**
  * @dev Contract that verifies that a set threshold of validators from a given blockchainID
- * have signed an off-chain Warp message and forwards the payload to the destination contract specified in the payload.
+ * have signed an off-chain Warp message and forwards the payload to the target contract specified in the payload.
```

Comment:
```suggestion
 * have signed an off-chain Warp message and forwards the payload to the target contract specified in the ValidatorSetSigMessage.
```
Reword to avoid confusion over the various payloads involved.